src/kaltura-player.ts

```diff
@@ -832,15 +836,17 @@ export class KalturaPlayer extends FakeEventTarget {
   }
 
   private _addAudioPreset(options: Partial<KalturaPlayerConfig>): void {
-    if (options.plugins?.audioPlayer) {
-      if (!window.kalturaCustomPresetMap) {
-        window.kalturaCustomPresetMap = {};
-      }
-      if (!window.kalturaCustomPresetMap[options.ui!.targetId]) {
-        window.kalturaCustomPresetMap[options.ui!.targetId] = {};
+    Object.keys(options.plugins).forEach((plugin) => {
+      if (window.kalturaCustomPreset[plugin]) {
+        if (!window.kalturaCustomPresetMap) {
+          window.kalturaCustomPresetMap = {};
+        }
+        if (!window.kalturaCustomPresetMap[options.ui!.targetId]) {
```

Comment:
you can shorten to something like

window.kalturaCustomPresetMap[this.config.targetId] = window.kalturaCustomPresetMap[this.config.targetId] || {};

Reply:
done

Reply:
checked again, this is not correct, in case we still don't have this player on kalturaCustomPresetMap we will get an error on the assignment:
![image](https://github.com/user-attachments/assets/ab73f200-c421-4be2-8099-afcf4b180688)
![image](https://github.com/user-attachments/assets/0c9a689e-e035-4626-a657-58a56077c240)
I need the complete check, i can short it to:
`if (window.kalturaCustomPreset?.[plugin]) {
    window.kalturaCustomPresetMap ??= {};
    window.kalturaCustomPresetMap[options.ui!.targetId] ??= {};
    window.kalturaCustomPresetMap[options.ui!.targetId][plugin] = window.kalturaCustomPreset[plugin];
}
`
but i'm not sure if it's a clear code.
so up to you